### PR TITLE
placeholder documentation for examples

### PR DESCRIPTION
**Pull request by nstauff**
Added preliminary README and CHANGELOG for WATTS

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [0.1.0] - not released yet
```
Comment (paulromano) [inline CHANGELOG.md]: It looks like in the "Keep a Changelog" site, they suggest making this
```suggestion
## [Unreleased]
```
and then moving it to a version once you make a release

File: examples/example6_adv_multiapps/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Example6 - Advance MOOSE MultiApp calculation
```
Comment (paulromano) [inline examples/example6_adv_multiapps/README.md:1]:
```suggestion
# Example6 - Advanced MOOSE MultiApp calculation
```

**Comment by nstauff** [comment]
@paulromano thanks! Feel free to approve the PR.

**Comment by paulromano** [comment]
@nstauff I'm a little confused -- it looks like you added much of the code from the pyarc PR to this one. I'd prefer to keep them separate -- one PR adding the CHANGELOG and example READMEs, and then the pyarc PR (along with its associated documentation).

**Comment by nstauff** [comment]
@paulromano I am confused too. I guess this is what happens when I have two PR from my Fork on the same development branch (I had kept my development branch on my Fork up-to-date, which may have been my mistake). I am not sure how to revert that.

**Review by paulromano — APPROVED** [review]

**Pull request merged**